dotnet-templates.sln

```diff
@@ -80,6 +80,10 @@ EndProjectSection
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Equinox.Templates", "src\Equinox.Templates\Equinox.Templates.fsproj", "{8C92B728-85A5-4231-863A-E4236E46CC36}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "equinox-shipment", "equinox-shipment", "{DAE9E2B9-EDA2-4064-B0CE-FD5294549374}"
```

Comment:
shipment -> shipping ?
i.e. make it more a noun?

Reply:
I'll rename thanks!